tests/wp-cldr-tests.php

```diff
@@ -219,4 +221,77 @@ public function test_first_day_of_week() {
 		$this->assertEquals( '', $this->cldr->first_day_of_week( 'bad-slug' ) );
 		$this->assertEquals( '', $this->cldr->first_day_of_week( '' ) );
 	}
+
+	public function test_get_currency_for_all_countries() {
+
+		$this->assertEquals( 'USD', $this->cldr->get_currency_for_all_countries()['US'] );
```

Comment:
I experimented with Circle CI but haven't tweaked its configuration yet. I think this might be a PHP version issue. The Travis tests are running fine so I disabled Circle CI for now.